src/commands/transaction/accountoperationrestriction.ts

```diff
@@ -0,0 +1,165 @@
+import { command, metadata, option } from 'clime';
+import {
+    Account,
+    AccountRestrictionModification,
+    AccountRestrictionModificationAction,
+    AccountRestrictionTransaction,
+    AccountRestrictionType,
+    Deadline,
+    TransactionHttp,
+    TransactionType,
+    UInt64,
+} from 'nem2-sdk';
+import { OptionsResolver } from '../../options-resolver';
+import { ProfileCommand, ProfileOptions } from '../../profile.command';
+import { ModificationActionValidator } from '../../validators/modificationAction.validator';
+import { RestrictionTypeValidator } from '../../validators/restrictionType.validator';
+
+export class CommandOptions extends ProfileOptions {
+    public static transactionType = ['RESERVED', 'TRANSFER', 'REGISTER_NAMESPACE', 'ADDRESS_ALIAS', 'MOSAIC_ALIAS', 'MOSAIC_DEFINITION',
```

Comment:
We might want to use the names from TransactionType class and do something like TransactionType['KEY'] instead of redefining all the transaction types.